crates/tests/src/e2e/ibc_tests.rs

```diff
@@ -2202,7 +2202,7 @@ fn check_tx_height(test: &Test, client: &mut NamadaCmd) -> Result<u32> {
         .split_once(' ')
         .unwrap()
         .1
-        .split_once('.')
+        .split_once(',')
```

Comment:
pls also change the full stop in the comment above `// Expecting e.g. "height 1337."`

Reply:
Done in 16a2c585a240bde7cb6fb6b9a2cc5e11fee03261